main/nginx/APKBUILD

```diff
@@ -304,7 +306,7 @@ _module() {
 	sonames="${sonames:-"ngx_${name}_module.so"}"
 
 	pkgdesc="$pkgdesc (module $name)"
-	depends="$pkgname $(eval "echo \$_${name}_depends")"
+	depends="$_provide $(eval "echo \$_${name}_depends")"
```

Comment:
@jirutka what do you think about this? both `openresty` and `nginx` will provide `cmd:nginx` but there is no guarantee which implementation the module will pull in.